settings.go

```diff
@@ -120,15 +128,17 @@ func GetLogFileOrDefault(useThisFile string) string {
 		return ResolvedLogFile
 	}
 
-	r := os.Getenv(clogFileEnv)
+	// start by preferring the file given to us by the caller.
+	r := useThisFile
 
-	// if no env var is specified, fall back to the default file location.
+	// if no file was provided, look for the default ENVj.
```

Comment:
😱 I thought I only had to worry about 't's

```suggestion
	// if no file was provided, look for the default ENV.
```

Reply:
What?  It's not even the same keyboard!

is it... is it me?

![on no](https://i.giphy.com/3o85xwSX5lrMO7B5e0.webp)